Packs/FeedCyrenThreatInDepth/Integrations/CyrenThreatInDepth/CyrenThreatInDepth.py

```diff
@@ -93,8 +93,11 @@ def to_indicator_objects(self) -> List[Dict]:
                                                     entitycategory=relationship.get("related_entity_category", "")))
             fields["cyrenfeedrelationships"] = relationship_indicators
 
+        raw_json = self.entry.copy()
+        raw_json["source_tag"] = FeedSource.PRIMARY
+        raw_json["tags"] = self.get_tags()
```

Comment:
Because of field differences between primary and related indicators and because we want to include multiple values from multiple fields into the final `tags` field, we decided to go with these intermediate fields. They help and prepare data for the mapper.